test/flutter_test_config.dart

```diff
@@ -3,11 +3,13 @@ import 'dart:async';
 import 'package:alchemist/alchemist.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_dsfr/flutter_dsfr.dart';
+import 'package:flutter_dsfr/src/theme/icons.dart';
```

Comment:
Can't you import everything with just `import 'package:flutter_dsfr/flutter_dsfr.dart';` ?